tests/keras/wrappers/scikit_learn_test.py

```diff
@@ -131,6 +131,7 @@ def test_regression_build_fn():
     assert_regression_works(reg)
 
 
+@pytest.mark.timeout(50)
```

Comment:
Why this test specifically?

Reply:
Please see the failed builds that I mentioned (links in related issues). Every time travis gets a 10 minutes timeout, it's on this test specifically. No idea why and I was hoping that this manual timeout would help us figure out what was happening